JavaScriptResource.mustache

```diff
@@ -75,27 +75,45 @@ fiftyoneDegreesManager = function() {
         }
     }
 
-    // Get cookies with the '51D_' prefix that have been added to the request
+    // Get 'cookies' with the '51D_' prefix that have been added to the request
```

Comment:
let's minimize the use of word 'cookie' - just say `storedValues` - just in case some static analyzer or some search tool would pick on it.